src/modules/bigquery.js

```diff
@@ -326,11 +326,13 @@ export default class BigQuery {
     }
     tables.forEach((target) => {
       const path = this.bfs(graph, table, target);
-      for (let i = 1; i < path.length; i++) {
-        const currTable = path[i];
-        const prevTable = path[i - 1];
-        const relationship = this.findRelationshipData(schema, currTable, prevTable);
-        relationships.push(relationship);
+      if (path != null) {
+        for (let i = 1; i < path.length; i++) {
+          const currTable = path[i];
+          const prevTable = path[i - 1];
+          const relationship = this.findRelationshipData(schema, currTable, prevTable);
+          relationships.push(relationship);
+        }
```

Comment:
I ran into an NPE here - I don't totally understand why, but I suppose a null check doesn't hurt.

Reply:
Ah, I know.  I think it's possible to have the `relationships` response from a dataset or asset be undefined...which stinks.  Yeah, I think it's safer to make defend against it (e.g. 
```suggestion
      if (path != null) {
        for (let i = 1; i < path.length; i++) {
          const currTable = path[i];
          const prevTable = path[i - 1];
          const relationship = this.findRelationshipData(schema, currTable, prevTable);
          (relationships || []).push(relationship);
        }
```

Reply:
It was failing on the "path.length" call, so "path" was null.

Reply:
oh weird...